add a validation error for false template invitation email properties (server)

MaximilianKrambach opened the pull request:
The translations currently arrive messed up, and it seems nice to return some validation error feedback here.

FinnStutzenstein commented:
Ok, good to check for issues here. The reason why we got this error is another thing. I'll create an issue for the reason.

ostcar commented on openslides/users/models.py:
```diff
@@ -232,10 +232,20 @@ def __missing__(self, key):
-        message = message.format(**message_format)
+        try:
+            message = message.format(**message_format)
+        except KeyError as e:
```
Please use a longer variable name then `e`. For example `err`

ostcar commented on openslides/users/models.py:
```diff
@@ -232,10 +232,20 @@ def __missing__(self, key):
-        subject = subject.format(**subject_format)
+        try:
+            subject = subject.format(**subject_format)
+        except KeyError as e:
```
The same here

MaximilianKrambach commented:
I fixed the two short variable names and rebased it all on the current master (this time the commit is not that changing that a rebase would confuse anyone)
